src/peft/tuners/adalora/bnb.py

```diff
@@ -51,36 +51,35 @@ def __init__(
 
             init_lora_weights = kwargs.pop("init_lora_weights", True)
             self.update_layer(adapter_name, r, lora_alpha, lora_dropout, init_lora_weights)
-            self.active_adapter = adapter_name
+            self.active_adapters = [adapter_name]
```

Comment:
I think this change should not be there no? Maybe you can follow the same logic as lora?

Reply:
Done